packages/ui-link/src/Link/__tests__/Link.test.tsx

```diff
@@ -188,9 +188,8 @@ describe('<Link />', async () => {
 
   describe('with `as` prop', async () => {
     describe('with `onClick`', async () => {
-      const onClick = stub()
```

Comment:
we should not put code in the `describe` block. When I ran tests via WebStorm these threw an exception that was somehow swallowed when running via console.